README.md

```diff
@@ -6,11 +6,11 @@ Easily host entire web applications on a single AWS Lambda function using this [
 
 ### Features
 
-* Designed to make it easy to host pre-existing web frameworks (e.g. Express.js, Hapi) or any large web application on a single AWS Lambda Function.
-* Blazing Fast Uploads via AWS S3 Accelerated Transfer and Multi-Part.
-* Dependencies are automatically put in AWS Lambda Layers, reducing cold-start time and further reducing upload time.
-* Simple shim for receiving and responding to HTTP requests.
-* Supports specifying custom domains.
+- Designed to make it easy to host pre-existing web frameworks (e.g. Express.js, Hapi) or any large web application on a single AWS Lambda Function.
```

Comment:
I'm sorry for this, I notice that they were automatically changed by the vscode linter. That said, I notice another code inconsistencies in `serverless.types.js`, please let me know if you want me to create another PR for that.

Reply:
No worries 😊